packages/client/helpers/build.ts

```diff
@@ -112,12 +113,10 @@ void build([
   generatorBuildConfig,
   nodeRuntimeBuildConfig('binary'),
   nodeRuntimeBuildConfig('library'),
-  nodeRuntimeBuildConfig('data-proxy'),
```

Comment:
We don't have a special Data Proxy runtime anymore, instead Data Proxy is always included in either Library or Binary. That enables a seamless switch between engines just based on the URL contents.